app/assets/javascripts/latlong_map.coffee

```diff
@@ -0,0 +1,39 @@
+marker = null
+
+# fallback to PVTA if we can't geocode the incident's location
+PVTA = { lat: 42.105552, lng: -72.596511 }
+
+createMap = (mapSelector, latLng, placeInitialMarker) ->
+  map = new google.maps.Map mapSelector[0], { zoom: 15, center: latLng }
```

Comment:
In playing with the map, I noticed it had a similar problem noticed [here](https://github.com/umts/ParkingMap/commit/ab1c1b5f905ae8b27630944d8737054c30aa78aa). Satellite view tilts at close zoom levels and the pins appear to be in the wrong place. I recommend `tilt: 0` here too.

Reply:
Interesting. For me it's always street view. Will do.